### PR TITLE
ja: update /api/pages-head

### PR DESCRIPTION
**Pull request by buntafujikawa**
fix https://github.com/vuejs-jp/ja.docs.nuxtjs/issues/292

@inouetakuya @aytdm
I fixed this issue. Please review it 🌈

### CODE REVIEW

File: ja/api/pages-head.md [hunk 1]
```diff
@@ -9,7 +9,8 @@ description: Nuxt.js はアプリケーションの headers 及び html attribut
 
 現在のページの HTML の head タグを設定するために `head` メソッドを使います。
 
-コンポーネントのデータは `head` メソッド内で `this` を使って利用できます。ページのデータを使って独自のメタタグを設定することもできます。
+コンポーネントのデータは `head` メソッド内で `this` を使って利用できます。ページのデータを使って独自のメタタグを設定することもできます。[Nuxt FAQ](https://nuxtjs.org/faq/) も必ずご覧ください。
+
```
Comment (aytdm) [inline ja/api/pages-head.md]: Would you remove this new line?

File: ja/api/pages-head.md [hunk 2]
```diff
@@ -9,7 +9,8 @@ description: Nuxt.js はアプリケーションの headers 及び html attribut
 
 現在のページの HTML の head タグを設定するために `head` メソッドを使います。
 
-コンポーネントのデータは `head` メソッド内で `this` を使って利用できます。ページのデータを使って独自のメタタグを設定することもできます。
+コンポーネントのデータは `head` メソッド内で `this` を使って利用できます。ページのデータを使って独自のメタタグを設定することもできます。[Nuxt FAQ](https://nuxtjs.org/faq/) も必ずご覧ください。
```
Comment (aytdm) [inline ja/api/pages-head.md]: I appreciate that replace the url japanese one.

```suggestion
コンポーネントのデータは `head` メソッド内で `this` を使って利用できます。ページのデータを使って独自のメタタグを設定することもできます。[Nuxt FAQ](https://ja.nuxtjs.org/faq/) も必ずご覧ください。
```

File: ja/api/pages-head.md (hunk 1 above)
Comment (buntafujikawa) [reply ja/api/pages-head.md]: 🙇

**Comment by buntafujikawa** [comment]
@aytdm
Thank you for your feedback, and please review it again 😄

**Pull request merged**